app/main/views/suppliers.py

```diff
@@ -25,6 +25,33 @@ def find_suppliers():
     )
 
 
+@main.route('/suppliers/<string:supplier_id>/edit/name', methods=['GET'])
+@login_required
+@role_required('admin', 'admin-ccs-category')
+def edit_supplier_name(supplier_id):
+
+    supplier = get_supplier(supplier_id)
+
+    return render_template(
+        "edit_supplier_name.html",
+        supplier=supplier["suppliers"],
+        **get_template_data()
+    )
+
+
+@main.route('/suppliers/<string:supplier_id>/edit/name', methods=['POST'])
```

Comment:
@TheDoubleK Well spotted—removed for debugging and restored in 55d7647 (which also addresses the `admin-ccs-category` thing